core/components/atoms/breadcrumb/breadcrumb.js

```diff
@@ -17,8 +18,9 @@ const LinkIcon = styled(Icon)`
 `
 
 const Wrapper = styled.div`
-  ${/* overrides for link */ ''};
+  ${containerStyles};
 
+  ${/* overrides for link */ ''};
```

Comment:
Why do we need this empty string to be inserted here?
```suggestion
  /* overrides for link */
```